app/src/sharedTest/java/org/oppia/android/app/profile/AdminPinActivityTest.kt

```diff
@@ -1011,6 +1028,12 @@ class AdminPinActivityTest {
     ApplicationProvider.getApplicationContext<TestApplication>().inject(this)
   }
 
+  private fun createAdminPinActivityIntent(): Intent {
+    return AdminPinActivity.createAdminPinActivityIntent(
+      ApplicationProvider.getApplicationContext(),0,-10710042,0
```

Comment:
Why do we need this? 
```
0,-10710042,0
```

Reply:
@anandwana001  The parameters for the method createAdminPinActivityIntent() were context, profileId, colorRgb and adminPinEnum so, 0,-10710042,0 were the default values which were used respectively for the other tests as well in the AdminPinActivityTest.kt.

Reply:
@anandwana001 @rt4914 please help me resolve this requested change.

Reply:
@starboi02 Let's wait for @anandwana001's reply, most probably it is correct, you just need to convert these to named arguments. Also, I suggest setting up the project completely by following the instructions in [wiki](https://github.com/oppia/oppia-android/wiki#installation) so that you don't get failed lint checks.

Reply:
@prayutsu sure and thanks for the link!

Reply:
Suggest removing this function and directly calling intent where you are using this function:

```
    activityTestRule.launchActivity(
      AdminPinActivity.createAdminPinActivityIntent(
        context = context,
        profileId = 0,
        colorRgb = -10710042,
        adminPinEnum = 0
      )
    )
```

This is suggested because the function that you are introducing should be used in all test cases which means it can be done in a separate PR.

Reply:
@starboi02 Please reply to this comment